packages/cli/src/commands/setup/deploy/providers/coherence.js

```diff
@@ -0,0 +1,112 @@
+// import terminalLink from 'terminal-link'
+import fs from 'fs'
+import path from 'path'
+
+import { getSchema, getConfig } from '@prisma/internals'
+import { Listr } from 'listr2'
+
+import { errorTelemetry } from '@redwoodjs/telemetry'
+
+import { getPaths, writeFilesTask } from '../../../../lib'
+import c from '../../../../lib/colors'
+import { printSetupNotes, addFilesTask } from '../helpers'
+import {
+  COHERENCE_HEALTH_CHECK,
+  DATABASE_YAML,
+  COHERENCE_YAML,
+} from '../templates/coherence'
+
+export const command = 'coherence'
+export const description = 'Setup Coherence deploy'
+
+export const getCoherenceYamlContent = async () => {
+  if (!fs.existsSync('api/db/schema.prisma')) {
```

Comment:
You can just use `getPaths()` here as well:

```suggestion
  if (!fs.existsSync(getPaths().api.dbSchema)) {
```